app/django_notipus/settings.py

```diff
@@ -36,8 +38,7 @@
 DEBUG = os.getenv("DEBUG", False)
 
 APP_NAME = os.environ.get("FLY_APP_NAME")
-# ALLOWED_HOSTS = [f"{APP_NAME}.fly.dev"]
-ALLOWED_HOSTS = ["*"]
+ALLOWED_HOSTS = [f"{APP_NAME}.fly.dev"]
```

Comment:
No, we need `*` here because otherwise we wouldn't be able to serve it from the notipus domain.